src/client/store/actions/audioActions.spec.js

```diff
@@ -0,0 +1,81 @@
+import {
```

Comment:
why file naming is spec, not test?

Reply:
such names were in tutorials, renamed in *.test.js